jormungandr-lib/src/interfaces/block0_configuration/DOCUMENTED_EXAMPLE.yaml

```diff
@@ -118,26 +118,37 @@ blockchain_configuration:
   # Halving formula: constant * ratio ^ (#epoch after epoch_start / epoch_rate)
   #
   reward_parameters:
-    halving: # or use "linear" for the linear formula
+    linear: # or use "linear" for the linear formula
       # In the linear formula, it represents the starting point of the contribution
       # at #epoch=0, whereas in halving formula is used as starting constant for
       # the calculation.
-      constant: 100
+      constant: 3835616440000
 
       # In the halving formula, an effective value between 0.0 to 1.0 indicates a
       # reducing contribution, whereas above 1.0 it indicate an acceleration of contribution.
       #
       # However in linear formula the meaning is just a scaling factor for the epoch zone
       # (current_epoch - start_epoch / epoch_rate). Further requirement is that this ratio
       # is expressed in fractional form (e.g. 1/2), which allow calculation in integer form.
-      ratio: "13/19"
+      ratio: "1/1"
 
       # indicates when this contribution start. note that if the epoch is not
       # the same or after the epoch_start, the overall contribution is zero.
       epoch_start: 1
 
       # the rate at which the contribution is tweaked related to epoch.
-      epoch_rate: 3
+      epoch_rate: 1
```

Comment:
I don't understand why you needed to change this value?

Reply:
The rate is not actually declining, so it makes more sense to set this to 1 than 3

Reply:
this is a `DOCUMENTED_EXAMPLE.md` not the settings for the testnet.

Reply:
See above.